nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -2688,6 +2688,12 @@ cp /var/lib/redis/dump.rdb &quot;/var/lib/redis-mastodon/dump.rdb&quot;
           enabled.
         </para>
       </listitem>
+      <listitem>
```

Comment:
Should probably go into the 22.11 release notes now, if at all.

Reply:
moved to 22.11 release notes